picard/ui/options/dialog.py

```diff
@@ -206,21 +200,13 @@ def load_all_pages(self):
                 log.exception("Failed loading options page %r", page)
                 self.disable_page(page.NAME)
 
-    def page_option_group(self, page):
-        try:
-            name = page.PARENT if page.PARENT in UserProfileGroups.SETTINGS_GROUPS else page.NAME
-            return UserProfileGroups.SETTINGS_GROUPS[name]
-        except (AttributeError, KeyError):
-            pass
-        return None
-
     def show_attached_profiles_dialog(self):
         window_title = _("Profiles Attached to Options")
         items = self.ui.pages_tree.selectedItems()
         if not items:
             return
         page = self.item_to_page[items[0]]
-        option_group = self.page_option_group(page)
+        option_group = UserProfileGroups.group_from_page(page)
```

Comment:
Yes, but I have the feeling `UserProfileGroups` can be totally removed and just use module level methods:

```python
def user_profile_group_from_page(page):
```

Instead of:
```python
class UserProfileGroups:

    @classmethod
    def group_from_page(cls, page):
```

This way we can only import methods we actually use:

```python
from picard.profile import user_profile_group_from_page
```

Current way is more or less equivalent to:

```python
from picard.profile import *
```



What do you think?

Reply:
Yes, I agree that's the case and would provide the nicer API

Reply:
See https://github.com/metabrainz/picard/pull/2433/commits/34b046fe31b55418352044b45c691ae6f53f3d78